### DISCUSSION TITLE
Feel the need... The need for speed is the tagline for which of this year’s Best Picture nominees?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#288] [Galaxy Trivia] [answered]
**Hint:** ✈️🔫

**Comment by thuongtruong1009** [comment]
"Feel the need... The need for speed" is a famous quote from the movie Top Gun, which was released in 1986 and was not nominated for Best Picture that year. It is possible that the quote was used in a different context in one of the current year's Best Picture nominees, but I do not have information on that.

**Comment by ItsNotAlexy** [comment, marked as answer]
The quote is from a movie called "Top Gun"!

**Reply by anggerpr** [reply]
Which top gun?

**Reply by ItsNotAlexy** [reply]
I believe top gun, episode 21 if im not mistaken

**Reply by ParanoidUser** [reply]
Bingo! One more to go for the `Galaxy Brain` badge. 😊
